ci_framework/roles/ci_multus/templates/nad.yml.j2

```diff
@@ -0,0 +1,24 @@
+{% for nad in _cifmw_ci_multus_nad_list %}
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  labels:
+    osp/net: {{ nad.name }}
+  name: {{ nad.name }}
+  namespace: {{ cifmw_ci_multus_namespace }}
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
```

Comment:
Can we store `cniVersion` in a Ansible var? It will be useful for testing different cni version in future?

Reply:
yeah, makes sense, I will add in the next update.